run-tests.py

```diff
@@ -37,4 +46,6 @@
 dir_safe_all_platforms = [os.path.join(os.path.dirname(__file__), x) for x in TESTS_ALL_PLATFORMS]
 dir_safe_non_rhel5 = [os.path.join(os.path.dirname(__file__), x) for x in TESTS_NON_RHEL5]
 
-sys.exit(run_tests(PACKAGES, dir_safe_all_platforms, dir_safe_non_rhel5))
+tests_exit_code = run_tests(PACKAGES, dir_safe_all_platforms, dir_safe_non_rhel5)
+
+sys.exit(flake8_exit_code or tests_exit_code)
```

Comment:
On 01/02/2015 04:19 PM, Chris Duryee wrote:
> cool!

Thanks! Could you LGTM this for me?